data/homepage.yaml

```diff
@@ -4,6 +4,30 @@ banner:
   pip_install: pip install ansible
   latest_version: Ansible 7 is now available.
   how_to_install: Install by copying the pip command.
+automate:
+  title: Why Ansible?
+  intro: Working in IT, you're likely doing the same tasks over and over. What if you could solve problems once and then automate your solutions going forward? Ansible is here to help.
+  top_section:
+    heading: Complexity kills productivity.
+    paragraphs:
+      automation: >-
+        Every business is a digital business.
+        Technology is your innovation engine, and delivering your applications faster helps you win.
+        Historically, that required a lot of manual effort and complicated coordination.
+        But today, there is Ansible - the simple, yet powerful IT automation engine that thousands of companies are using to drive complexity out of their environments and accelerate DevOps initiatives.
+  bottom_section:
+    heading: Ansible loves the repetitive work your people hate.
+    paragraphs:
+      time: >-
+        No one likes repetitive tasks.
+        With Ansible, IT admins can begin automating away the drudgery from their daily tasks.
+        Automation frees admins up to focus on efforts that help deliver more value to the business by speeding time to application delivery, and building on a culture of success.
+        Ultimately, Ansible gives teams the one thing they can never get enough of: time.
+        Allowing smart people to focus on smart things.
+      simple: >-
+        Ansible is a simple automation language that can perfectly describe an IT application infrastructure.
+        It's easy-to-learn, self-documenting, and doesn't require a grad-level computer science degree to read.
```

Comment:
grad-level -> graduate-level

Reply:
@cybette Thanks for the review. I can fix this but I copied the text verbatim from the wireframe. I'd like to revisit this and invite some of the community to review it as well. Not sure of the origins of this text but I feel like it leans a little too heavy towards "marketing" language for business use cases.

Honestly I also find this particular sentence to be somewhat pandering in its tone. While I agree with the overall sentiment here I think we can just say it better.

In fact I might try to give it a pass now. @leogallego Your thoughts here would be super valuable too. Any suggestions on improvement? Thanks.

Reply:
I agree, while it's not a sales pitch, it is somewhat marketing speak. I was doing more of a grammar/typo check for now since I expect we will probably revisit this :)

Reply:
Thanks, I just took another stab at it. Also realized we should probably add a grammar / spell checker. I'll create an issue for that.